sdk/src/main/java/io/opentelemetry/sdk/trace/export/BatchSpansProcessor.java

```diff
@@ -267,12 +272,25 @@ private Worker(
       this.maxExportBatchSize = maxExportBatchSize;
       this.spansList = new ArrayList<>(maxQueueSize);
       this.exporterTimeoutMillis = exporterTimeoutMillis;
+
+      Meter meter = OpenTelemetry.getMeterRegistry().get("opentelemetry");
+      LongCounter droppedSpansCounter =
+          meter
+              .longCounterBuilder("droppedSpans")
+              .setMonotonic(true)
+              .setUnit("1")
+              .setDescription(
+                  "The number of spans dropped by the BatchSpansProcessor due to high throughput.")
+              .build();
+      this.droppedSpans =
```

Comment:
As a limitation of the current API for metrics the bound needs to be static or you need to `unbind` it, maybe in shutdown?

Also creating the same instrument twice may fail (we have not implement that check yet). So the instrument can easily be static :)

Reply:
Making it static seems like a good option. I'll do it.

Reply:
ok, done! Thanks for the quick feedback on this one.